flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/optimize/JoinHintResolver.java

```diff
@@ -99,10 +105,14 @@ private RelNode visitBiRel(BiRel biRel) {
                 assert null != lookupTable;
                 if (rightName.isPresent() && matchIdentifier(lookupTable, rightName.get())) {
                     validHints.add(trimInheritPath(hint));
+                    updateInfoForOptionCheck(hint.hintName, leftName);
```

Comment:
here should be rightName instead?

Reply:
Actually, there is no need to check multi table name for lookup join because there only is one table for lookup join hint. So I delete this part.

Reply:
done